src/ObjectUtils/include/ObjectUtils/CoffFile.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef OBJECT_UTILS_COFF_FILE_H_
+#define OBJECT_UTILS_COFF_FILE_H_
+
+#include <filesystem>
+#include <memory>
+
+#include "ObjectUtils/ObjectFile.h"
+#include "OrbitBase/Result.h"
+#include "llvm/Object/Binary.h"
+#include "llvm/Object/ObjectFile.h"
+#include "symbol.pb.h"
+
+namespace orbit_object_utils {
+
+class CoffFile : public ObjectFile {
```

Comment:
I guess you opted for having `CoffFile` and `CoffFileImpl` for symmetry reasons with `ElfFile`?

I personally find it simpler to put the implementation in `CoffFile` directly, but I also don't feel strongly about that.

Reply:
Yes, it's purely for symmetry. I don't feel strongly either way, so I'll keep it as-is for now.